SqlScriptDom/Parser/TSql/TSqlParser.cs

```diff
@@ -60,17 +60,28 @@ public TSqlParser Create(SqlVersion tsqlParserVersion, bool initialQuotedIdentif
             }
         }
 
+        [Obsolete("This method is obsolete. Call TSqlParser.CreateParser instead.")]
```

Comment:
Does the build warning go away if the Obsolete attribute is on the line right above the function signature?